src/librbd/image/CloneRequest.h

```diff
@@ -67,21 +67,10 @@ class CloneRequest {
    *    v                                 |
    * SET PARENT * * * * * * * * * * > CLOSE CHILD
    *    |                               ^
-   *    |\--------\                     *
-   *    |         |                     *
-   *    |         v (clone v2 disabled) *
-   *    |     V1 ADD CHILD  * * * * * * ^
-   *    |         |                     *
-   *    |         v                     *
-   *    |     V1 VALIDATE PROTECTED * * ^
-   *    |         |                     *
-   *    v         |                     *
-   * V2 SET CLONE * * * * * * * * * * * ^
-   *    |         |                     *
-   *    v         |                     *
-   * V2 ATTACH CHILD  * * * * * * * * * *
-   *    |         |                     *
-   *    v         v                     *
+   *    v                               *
+   * SET CHILD  * * * * * * * * * * * * *
```

Comment:
Nit: "attach child" to mirror method names (and external state machine name)

Reply:
Ok, then I will change `SET PARENT` to `ATTACH PARENT`, because the reason why I used `SET CHILD` was to looks similar to what we already had.